### PR TITLE
Feature/weeereusedoffsite

### PR DESCRIPTION
**Pull request by JB12298**

### CODE REVIEW

File: src/EA.Weee.Core.Tests.Unit/AatfReturn/AddressDataTests.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    using System;
+    using System.ComponentModel.DataAnnotations;
+    using EA.Weee.Core.AatfReturn;
+    using FluentAssertions;
+    using Xunit;
+
+    public class AddressDataTests
+    {
+        [Theory]
+        [InlineData("Name")]
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Core.Tests.Unit/AatfReturn/AddressDataTests.cs]: Can also add tests for the max length attribute

File: src/EA.Weee.Web/Areas/AATFReturn/ViewModels/ReusedOffSiteViewModel.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    public class ReusedOffSiteViewModel : RadioButtonStringCollectionViewModel
+    {
+        public Guid OrganisationId { get; set; }
+
+        public Guid ReturnId { get; set; }
+
+        public Guid AatfId { get; set; }
+
+        [Required(ErrorMessage = "You must tell us whether any of this WEEE has gone directly to another site to be reused")]
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Areas/AATFReturn/ViewModels/ReusedOffSiteViewModel.cs]: Add simple unit test to ensure the require field attribute is present

Comment (JB12298) [reply src/EA.Weee.Web/Areas/AATFReturn/ViewModels/ReusedOffSiteViewModel.cs]: I think this is covered in ReusedOffSiteViewModelTests.cs?

File: src/EA.Weee.Core.Tests.Unit/AatfReturn/AddressDataTests.cs (hunk 1 above)
Comment (JB12298) [reply src/EA.Weee.Core.Tests.Unit/AatfReturn/AddressDataTests.cs]: Added

**Pull request merged**